README.md

```diff
@@ -11,7 +11,7 @@
 
 #### What?
 
-The Project Week is a week-long hackathon of hands on activity in which medical image computing researchers create solutions using the open source image computing platform, [3D Slicer](http://www.slicer.org), and [VTK](http://www.vtk.org), [ITK](http://www.itk.org), [CMake](http://www.cmake.org), and [CDash](http://www.cdash.org) libraries. Participants work collaboratively on solutions that lie on the interfaces of the fields of computer science, mechanical engineering, biomedical engineering, and medicine. In contrast to conferences and workshops where the primary focus is to report results, the objective of the Project Week is to provide a venue for creators of medical image computing open-source software creators to collaboratively work.
+The Project Week is a week-long hackathon of hands on activity in which medical image computing researchers create solutions using the open source image computing platform, [3D Slicer](http://www.slicer.org), and [VTK](http://www.vtk.org), [ITK](http://www.itk.org), [CMake](http://www.cmake.org), and [CDash](http://www.cdash.org) libraries as well as [OHIF](http://ohif.org), [vtkjs](https://kitware.github.io/vtk-js/), [Cornerstone](https://cornerstonejs.org/), and related web technologies. Participants work collaboratively on solutions that lie on the interfaces of the fields of computer science, mechanical engineering, biomedical engineering, and medicine. In contrast to conferences and workshops where the primary focus is to report results, the objective of the Project Week is to provide a venue for creators of medical image computing open-source software creators to collaboratively work.
```

Comment:
:+1: 

What about adding `[itkjs](https://insightsoftwareconsortium.github.io/itk-js/)` and `[Girder](https://girder.readthedocs.io/)` ?